cli/definitions/cli.ts

```diff
@@ -137,12 +137,14 @@ export interface IPatchCommand extends ICommand, IPackageInfo {
     appName: string;
     deploymentName: string;
     label: string;
+    appStoreVersion?: string;
```

Comment:
It's a shame that this (and below) is `appStoreVersion` and not `appVersion` to match `PackageInfo.appVersion` or `targetBinaryVersion` to match the command option.

Fewer different names for the same thing is a good thing.